apps/app-frontend/src/store/theme.js

```diff
@@ -17,7 +17,15 @@ export const useTheming = defineStore('themeStore', {
       for (const theme of this.themeOptions) {
         document.getElementsByTagName('html')[0].classList.remove(`${theme}-mode`)
       }
-      document.getElementsByTagName('html')[0].classList.add(`${this.selectedTheme}-mode`)
+      let theme = this.selectedTheme
+      if (theme === 'system') {
+        const prefersDark = window.matchMedia('(prefers-color-scheme: dark)')
```

Comment:
`window.matchMedia('(prefers-color-scheme: dark)')` always returns true for me. Maybe use [Tauri's api](https://v2.tauri.app/reference/javascript/api/namespacewindow/#example-29). It has methods for getting current theme & adding event listener for it.